Add conicConformalNetherlands with tests for it and some minor other things

JulesBlm opened the pull request:
I've added a composite projection for the Netherlands that includes its special municipalities in the Caribbean: Bonaire, Sint Eustatius and Saba. [You can view it here on Observable](https://observablehq.com/@julesblm/the-netherlands-composite-projection). Tests for the projection are included. Additionally, I have updated most dependencies and improved the formatting of the README.md. All tests pass.

Best,
Jules

JulesBlm committed:
Add conicConformalNetherlands and tests for it, update dependencies and improve formatting of README.md

rveciana reviewed (approved):
Thanks a lot for the new projection and the corrections. 
Once it's merged, can you adapt your observable to use the regular import?
I'm willing to see some election map for the Netherlands using your projection! I never found one I had confidence with to add it...

JulesBlm commented:
Thanks for the quick merge. I've updated my Observable notebook to import it from here. I've also added the new projection in your Observable notebook on d3-composite-projections and made a suggestion (Observable's way of a pull request) to that notebook.
